### DISCUSSION TITLE
Demo for frontend toolkit/frameworks

### DISCUSSION THREAD

**Discussion by adaniello** [add2cal/add-to-calendar-button#317] [Ideas / Requests]
Hi,
why not add demo using the most popular frontend toolkit/frameworks?

**Comment by jekuer** [comment]
You mean like the integration guides at https://add-to-calendar-button.com/#installation ?
Or having a repo with examples for all the frameworks?
Though about the latter one. Since it should be quite straightforward, I would not push for it at the moment, since it is mainly a lot of effort keeping the environments up-to-date, while is has barely any additional benefit - but maybe I am missing something.

What could be interesting: Starting a collection with external repositories making use of the button.

**Reply by adaniello** [reply]
I'm thinking about a repository with examples for all frameworks, eg. use case for Bootstrap, use case for Foundation, and so on
Just an idea.

**Comment by jekuer** [comment]
We set up a set of full project examples at Stackblitz.
There, we will add a growing number of different examples.

➡️ [https://stackblitz.com/@add-to-calendar/collections/add-to-calendar-button-integration-examples](https://stackblitz.com/@add-to-calendar/collections/add-to-calendar-button-integration-examples)
